examples/react-app/package.json

```diff
@@ -3,17 +3,19 @@
   "version": "1.0.0",
   "private": true,
   "dependencies": {
-    "@cmsgov/design-system": "^2.0.0",
-    "react": "^16.2.0",
-    "react-dom": "^16.2.0"
+    "@cmsgov/design-system": "file:../../packages/design-system/",
+    "react": "^16.14.0",
+    "react-dom": "^16.14.0"
   },
   "devDependencies": {
     "@babel/core": "^7.8.4",
     "@babel/preset-env": "^7.8.4",
     "@babel/preset-react": "^7.8.3",
     "babel-loader": "^8.0.0",
+    "css-loader": "5.2.7",
     "gulp": "^4.0.2",
     "gulp-sass": "^3.1.0",
+    "style-loader": "2.0.0",
```

Comment:
`css-loader` and `style-loader` were required for this project to properly render the imported CSS file from our design system. These specific versions were required as they're the last to work with our version of Webpack.

**tldr; these packages were added to import polyfill CSS.**